content/en/docs/releases/next-release-preview/index.md

```diff
@@ -80,3 +80,7 @@ tasks:
       max-number-of-pipeline-executions-to-process:
       execution-retrieval-timeout-seconds:
 ```
+
+### Clouddriver
```

Comment:
Mind making this more general, to cover the other services we've upgraded?

Reply:
done